src/exports.ts

```diff
@@ -155,8 +155,8 @@ export class ModelExports {
     subheading: 'Loading',
     useDocsFrom: 'loadModelInternal'
   })
-  static loadModel(modelConfigPath: string): Promise<Model> {
-    return loadModelInternal(modelConfigPath);
+  static loadModel(pathOrIOHandler: string|io.IOHandler): Promise<Model> {
```

Comment:
I think the design settled on is not backward compatible, so we'll need to make sure we bump the middle version number, right?  (While we still accept strings like before, we require a prefix that we didn't in the past)